### DISCUSSION TITLE
Media access is over http/a only.

### DISCUSSION THREAD

**Discussion by aykaramba** [rabbibotton/clog#36] [General]
I just wanted to get confirmation that CLOG only accesses resource files over http/s. I presume it is because of the following event chain: 

CLOG <-> js on client <-> online resource files like images, video, etc.

In other words, CLOG does not serve any media files to the client at all, correct?  There is no way to access local resources over nfs and hand them over to the js client?  

Note: I don't need local file access, http/s only probably makes most sense.  Just looking for clarification.

**Comment by rabbibotton** [comment]
I believe you can use a “file” url but haven’t played with it. However since CLOG is an http server and has full access to the OS etc not a big deal to give the http server access to anything you want or symbolic access on demand etc

**Reply by aykaramba** [reply]
Aha!  Thank you.

**Comment by aykaramba** [comment]
For any noobs wondering, these work:

(setf (background-image *body*) "~/path/path/file.png")
(setf (background-image *body*) "https://somewhere.url/wallpaper.jpg")

Not sure why the local path was not working for me earlier. 

Thanks for the tip Dr. Botton.

**Reply by rabbibotton** [reply]
Remember though local urls only work for local apps. Meaning if you deploy a clog app remotely and try and use at the remote location a local URL is going to look for the file at the remote location (provided security protocols don’t prohibit all together of the users remote browser) not in the server.
